docker/generic/start_proxy.py

```diff
@@ -489,6 +489,24 @@ def make_argparser():
     Default value: Not enabled. Please provide the certificate and key files
     /etc/nginx/ssl/backend.crt and /etc/nginx/ssl/backend.key.''')
 
+    # always_print_primitive_fields.
+    parser.add_argument('--transcoding_always_print_primitive_fields',
+                        action='store_true',
+                        help=argparse.SUPPRESS)
```

Comment:
My understanding is SUPRESS is to hide the flag.  I don't think we need to hide it